bolts/fair/data/datamodules/vision/celeba.py

```diff
@@ -4,26 +4,28 @@
 
 import ethicml as em
 import ethicml.vision as emvi
-from kit import implements
+from kit import implements, parsable
+from kit.torch import prop_random_split
 from pytorch_lightning import LightningDataModule
 import torch
 from torch.utils.data.dataset import random_split
-from torchvision import transforms as TF
+from torchvision import transforms as T
 
 from bolts.fair.data.datasets import TiWrapper
 
-from .base import VisionBaseDataModule
+from .base import BaseVisionDataModule
 
 __all__ = ["CelebaDataModule"]
 
 
-class CelebaDataModule(VisionBaseDataModule):
-    """CelebA Dataset."""
+class CelebaDataModule(BaseVisionDataModule):
+    """Fairness-oriented CelebA data-module."""
```

Comment:
Eventually this datamodule should use the CelebA dataset class I added, right?

Reply:
Yes but it's beyond the scope of this PR